### PR TITLE
change L1 DQM to have stage1 HI plots and change stage1 pp QTest

### PR DESCRIPTION
**Pull request by jingyucms**
This PR inherit from #11777 plus a fix for @slava77 's comment about making sure it can plot run1 data. 
Git prevent me just update #11777 with this change and I don't know how to make it to work, so I just override #11777 and make a new one.

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by mulhearn** [comment]
+1

**Comment by slava77** [comment]
I think it's a bad solution to run stage1 on run1 data.
Every event we now have
```
%MSG-e L1T:  L1TRawToDigi:caloStage1Digis 20-Nov-2015 00:10:47 CET  Run: 165121 Event: 23609118
Cannot unpack: empty/invalid L1T raw data (size = 0) for ID 1352. Returning empty collections!
%MSG
%MSG-e L1T:  RctRawToDigi:rctDigis 20-Nov-2015 00:10:47 CET  Run: 165121 Event: 23609118
Cannot unpack: empty/invalid L1T raw data (size = 0) for ID 1350. Returning empty collections!
```
these should be silenced or the irrelevant unpackers removed.

**Comment by slava77** [comment]
see, e.g, 
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12505/9871/runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log
in the latest tests using matrix workflow 1000.0

**Comment by slava77** [comment]
so, this still needs work, before it can go to the general release area

**Comment by jingyucms** [comment]
Ok...I can easily run 2 copies of the code, one for gct and one for stage1, so for run1 data, gct will run fine and stage1 will produce this emply collection error and for 2015 data, both runs. But to kill this kind of message, I have no idea how to do it at this moment. @slava77 Do you agree with the easy fix that just running in parallel?

**Comment by slava77** [comment]
@jingyucms 
maybe I need to see what you will do.
I'm expecting that there should be nothing upgrade trigger (stage-1/2) related running for run1 setup.

**Comment by slava77** [comment]
[silencing the error message is still an option though: an unpacker is supposed to try to unpack and if FEDs are not in place, to just make an empty output collection. An error or even warning about a missing FED should be made only if there is real certainty it has to be in place. This is a picture for the detector data. Trigger may be special.]

**Comment by mulhearn** [comment]
@jingyucms:  let's chat about strategy for this today.

**Comment by slava77** [comment]
@jingyucms @mulhearn 
are you planning an update here soon?

**Comment by jingyucms** [comment]
@slava77 Yes I am going to put an Eras to make sure it works for both run1 and run2. It gonna happen the latest tomorrow.

**Comment by jingyucms** [comment]
@slava77 @mulhearn @deguio Changed offline config to run Run1 as default while stage1 is available with eras and customise. The online config remains to be running parallel with Legacy and Stage1.
Also changed resolution for some of the HIon histogram.

**Comment by deguio** [comment]
please test

**Comment by slava77** [comment]
there you go, the merge conflicts are now "official"

**Comment by slava77** [comment]
@cmsbuild please test

File: L1Trigger/L1TCommon/python/customsPostLS1.py [hunk 1]
```diff
@@ -153,3 +153,44 @@ def customiseSimL1EmulatorForPostLS1_Additional_HI(process):
     if hasattr(process,'caloConfig'):
         process.caloConfig.fwVersionLayer2 = cms.uint32(1)
     return process
+
+# dustomization for offline DQM
+def customiseStage1ForOfflineDQM(process):
```
Comment (slava77) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: where is this function called?
75X is not based on eras
this function call probably fits in postLS1Customs.customise_DQM_25ns (if it's appropriate for both data and MC)
maybe even in customise_DQM

Comment (jingyucms) [inline L1Trigger/L1TCommon/python/customsPostLS1.py]: @slava77 This can goes to customise_DQM_25ns, it fits for data and MC.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by mulhearn** [comment]
+1

**Comment by jingyucms** [comment]
@slava77 I changed another thing make the data emulator comparison for run2 workflow looks good.  I have no plan to commit more things on my side. Please test again. Thanks for your patience.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by mulhearn** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by slava77** [comment]
@cmsbuild please test
the last update was somewhat trivial
the previous larger changes still need to be signed off by dqm @deguio @danduggan

**Comment by deguio** [comment]
+1
fine for me.

**Comment by civanch** [comment]
+1

**Comment by mulhearn** [comment]
+1

**Comment by jingyucms** [comment]
@slava77 Can this merge?

**Comment by slava77** [comment]
@jingyucms 
Is there a PR for 80X?
Is this PR needed for HI MC?

**Comment by jingyucms** [comment]
@slava77 These PR is not needed MC production. The 80X version of this PR is merged. #12574

**Comment by davidlange6** [comment]
why is it needed in 76x ? Future work will be in 80x.

**Comment by jingyucms** [comment]
@davidlange6 Both Online/offline dqm is need to run this one. This has a layout change and HIon change for the online dqm and stage1 customization for offline dqm.

**Comment by davidlange6** [comment]
Right, so 80x is perfect..

**Comment by jingyucms** [comment]
@davidlange6 Yes, but as we start data taking this year with 76x and 75x. If this PR can be back ported to 76x, it will me much more convenience for us.

**Pull request closed without merging**